pkg/controller/instance/instance_controller.go

```diff
@@ -33,6 +33,7 @@ import (
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Let's use `metav1` to make it clearer.
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```